### DISCUSSION TITLE
Implement ProjectcontourIoV1Api in python kubernetes-client library

### DISCUSSION THREAD

**Discussion by goldiee36** [projectcontour/contour#4974] [Ideas]
Hi,

I would like to ask if it's possible or is there any plan to add ProjectcontourIoV1Api in the Kubernetes python client?

https://github.com/kubernetes-client/python

Currently I did not find a way to create a HTTPProxy via python.

Thanks in advance
Adam

**Comment by sunjayBhatia** [comment]
We don't have plans at the moment to add a python client, I believe the client you link to is for upstream APIs and not project-specific CRs, so you may have to generate your own client or use the HTTP APIs directly

**Comment by goldiee36** [comment]
Thanks for the answer!
